### DISCUSSION TITLE
Wiki markdown not properly rendered

### DISCUSSION THREAD

**Discussion by mgfernan** [community/community#23143] [New to GitHub]
<p>We have the following wiki page</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/rokubun/jobs/wiki" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:256/256;"><img src="https://avatars.githubusercontent.com/u/11183336?s=400&amp;amp;v=4" class="thumbnail" width="256" height="256"></div>

<h3><a href="https://github.com/rokubun/jobs/wiki" target="_blank" rel="noopener nofollow ugc">rokubun/jobs</a></h3>

  <p><span class="label1">Job offers available at Rokubun. Contribute to rokubun/jobs development by creating an account on GitHub.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Which we populate using Markdown documents. Once we commit any change it works as expected but after a couple of reloads it ceases to render properly (unformatted text with the HTML tags).</p>
<p>Is it a bug? caused by a non-valid character? Have you experience this problem? Any hint on how to solve it would be greatly appreciated.</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>If this is only happening to you today, I have good news: It’s broken and we’re working on fixing it.</p>
<p>If it’s something that has been more long term, it’d be great to see some screenshots of the issue you’re seeing.</p>

**Comment by mgfernan** [comment]
<p>Hi, thanks for the quick reply. I cannot reproduce the error anymore so it seems that the bug is gone. Many thanks!</p>

**Comment by shiftkey** [comment, marked as answer]
<p>:wave: a fix for this has been deployed to help out others, but I’m glad this is resolved for you <a class="mention" href="/tryskele">@Tryskele</a>!</p>
